src/tiles/tile-dispatcher.ts

```diff
@@ -222,24 +288,45 @@ export class TileDispatcher
       }
 
       #container {
+        position: relative;
         height: 100%;
+        border-radius: 4px;
       }
 
-      #delete-button {
-        float: right;
+      #container.hoverable:hover {
+        box-shadow: 0 0 6px 2px rgba(8, 8, 32, 0.8);
+        transition: box-shadow 0.1s ease;
       }
 
       a {
         display: block;
         height: 100%;
         color: unset;
         text-decoration: none;
+        transition: transform 0.05s ease;
       }
 
       a :first-child {
         display: block;
         height: 100%;
       }
+
+      #touch-backdrop {
+        position: fixed;
+        width: 100vw;
+        height: 100vh;
+        top: 0;
+        left: 0;
+        z-index: 1;
+        background: transparent;
+      }
+
+      tile-hover-pane {
+        position: absolute;
+        top: -2000px;
+        left: -2000px;
```

Comment:
QUESTION: Is this value intended to take it off screen? If someone has a really big monitor, will these values still work?

Reply:
That was the intent, yeah -- these positions should generally only appear for a single tick before the popup is positioned correctly, so they're off-screen to avoid a visible flicker. But I bet there's some monitor/situation out there that breaks this assumption. I'll adjust this a bit to future-proof better.